platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js

```diff
@@ -1,4 +1,5 @@
 import { api } from 'dicomweb-client';
+import * as dcmjs from 'dcmjs';
```

Comment:
I thought we fixed this so you no longer need to import as `*`?

Reply:
Oh cool, I'm not sure I had the latest version on this branch